docker/main/rootfs/usr/local/nginx/conf/nginx.conf

```diff
@@ -222,12 +222,16 @@ http {
             include proxy.conf;
         }
 
-        location ~* /api/.*\.(jpg|jpeg|png|webp|gif)$ {
-            include auth_request.conf;
-            rewrite ^/api/(.*)$ $1 break;
-            proxy_pass http://frigate_api;
-            include proxy.conf;
-        }
+# FIXME: Needed to disabled this rule, otherwise it fails for endpoints that end with one of those file extensions
```

Comment:
Yes, me too. I tried to do some debugging and always ended up on the same line of code: https://github.com/encode/uvicorn/blob/47304d9ae76321f0f5f649ff4f73e09b17085933/uvicorn/protocols/http/httptools_impl.py#L165

If I invoked the API directly (i.e., localhost:5001 instead of localhost:5173/api...) it works just fine. 

If any of you can take a look I'd appreciate.

To have more verbose logs and debug the code further I did the following:

<details>
<summary>Details</summary>
<img width="902" alt="image" src="https://github.com/user-attachments/assets/87555d0f-8ede-4598-bca2-9ab25971f231">

<img width="478" alt="image" src="https://github.com/user-attachments/assets/eda0614c-6626-437e-a47f-d3297c5abce1">

<img width="381" alt="image" src="https://github.com/user-attachments/assets/b5d16b63-cd6f-4277-9425-ee57e9fe32c6">

</details>